src/main/java/com/openmrs/joiners/controller/PersonController.java

```diff
@@ -18,4 +20,9 @@ public class PersonController {
     public ResponseEntity<List<Person>> getAllPersons(){
         return ResponseEntity.ok(personService.getAllPersons());
     }
+
+    @PutMapping(value = "/update/department")
```

Comment:
@Timug-123 should this endpoint be named department yet we are working with the person class?  additionally, can you find a suitable return status for the method, rather than 200 (OK)?